dev/RELEASE_PROCESS.rst

```diff
@@ -18,17 +18,36 @@ This document describes the process for creating new releases.
    by the script above, but they aren't yet.
 
 4. **Increment the Python version:** Create a PR that increments the Python
-   package version. See `this example`_.
+   package version. See `this example`_. You can use the following to get started,
+   assuming you're on a clean master:
+
+   .. code-block:: bash
```

Comment:
The point of this block of code is to update all of the documentation? It might accidentally get too much stuff (e.g., version strings might appear in deprecation warnings). Also, it seems a bit excessive to tell people how to do a find and replace.. Let's just tell people to search for all instances of the previous version in the repository and replace all relevant ones with the new version number.

Also, let's separate this out as a different bullet point and tell people that it should be a separate PR from the version bump PR.

Reply:
No, this is _only_ a version bump PR.

Reply:
This seems too confusing and heavyweight.